activiti-spring-boot-starter/src/test/resources/processes/connector-var-mapping.bpmn20.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the name of this file is misleading...this json is a connector definition it's not a variable mapping file.

Reply:
I would suggest to change it for clarity